### DISCUSSION TITLE
UWP

### DISCUSSION THREAD

**Discussion by ericleroy01** [SuRGeoNix/Flyleaf#347] [General]
Hi, can I use your lib in an UWP app on XBOX and playing 4K video ?

**Comment by SuRGeoNix** [comment]
Hi @ericleroy01, never tried and not sure what would be required to run in UWP. To be honest no plans to support UWP in any case but might be possible to make it work with some small changes. If that's the case feel free to create a PR! Hopefully, MS will support WinUI 3 for XBOX soon.
